app/models/country.rb

```diff
@@ -1,4 +1,20 @@
 class Country < ActiveRecord::Base
-  belongs_to :language, :foreign_key => 'LanguageID'
+
+  # create_table :countries do |t|
```

Comment:
Cool.  I didn't know about the annotate_models gem.  That sounds like a great solution!